data/core/tips.json

```diff
@@ -257,7 +257,7 @@
   {
     "type": "snippet",
     "category": "tip",
-    "text": [ "Food's shelf life is not given.  Storage conditions matter, especially temperature." ]
+    "text": [ "Storage conditions can affect the shelf life of foods.  The colder, the better." ]
```

Comment:
As I think of it now, "storage conditions" can have many meanings, and might be misleading in this tip, as for now only temperature is the only condition directly affecting effective shelf life. Perhaps "storing temperature greatly affects the effective shelf life of food. Cooler is better."

Reply:
I'll re-word this one.